api/migrations/20250212090653-fix-status-youngs-cle07.js

```diff
@@ -0,0 +1,16 @@
+const { logger } = require("../src/logger");
+const { YOUNG_STATUS_PHASE1, YOUNG_STATUS } = require("snu-lib");
+const { YoungModel } = require("../src/models");
+
+module.exports = {
+  async up() {
+    const youngsInCohortCle07Mars2025Updated = await YoungModel.updateMany(
+      { status: YOUNG_STATUS.VALIDATED, statusPhase1: YOUNG_STATUS_PHASE1.WAITING_AFFECTATION, cohort: "2025 CLE 07 - Mars" },
```

Comment:
```suggestion
      { status: YOUNG_STATUS.VALIDATED, statusPhase1: YOUNG_STATUS_PHASE1.WAITING_AFFECTATION, cohort: "2025 CLE 07 - Mars", ligneId: { $exists: true }, cohesionCenterId: { $exists: true } },
```
attention, là tu prend tout le monde même ceux qui n'avait pas encore été affecté, il faut s'assurer qu'on leur avait bien affecté un centre et une ligne de bus !

Reply:
93 jeunes n'ont pas de données d'affectation sur cette cohort